README.md

```diff
@@ -26,6 +28,11 @@ Team in the Inaugural DIHARD Challenge](https://www.isca-speech.org/archive/Inte
 * [Speaker diarization using convolutional neural network for statistics accumulation refinement](https://pdfs.semanticscholar.org/35c4/0fde977932d8a3cd24f5a1724c9dbca8b38d.pdf)
 * [pyannote. metrics: a toolkit for reproducible evaluation, diagnostic, and error analysis of speaker diarization systems](https://www.isca-speech.org/archive/Interspeech_2017/pdfs/0411.PDF)
 * [Speaker Change Detection in Broadcast TV using Bidirectional Long Short-Term Memory Networks](https://pdfs.semanticscholar.org/edff/b62b32ffcc2b5cc846e26375cb300fac9ecc.pdf)
+* [Speaker2Vec: Unsupervised Learning and Adaptation of a Speaker Manifold using Deep Neural Networks with an Evaluation on Speaker Segmentation](http://scuba.usc.edu/pdf/jati2017_Speaker2Vec.pdf)
+* [Speaker Diarization using Deep Recurrent Convolutional Neural Networks for Speaker Embeddings](https://arxiv.org/abs/1708.02840)
+* [Deep Speaker: an End-to-End Neural Speaker Embedding System](https://arxiv.org/abs/1705.02304v1)
```

Comment:
This one as well. It's focus on speaker recognition. The connection with diarization is implicit, thus prefer not including it here.

Reply:
I agree with you. Sorry about that. Thanks for your interest. :)

Reply:
The Deep Speaker and Speaker2Vec papers are still there. Please remove these two and commit/push again to your own branch, then I will merge the pull request.

Reply:
I have deleted those papers. 👍